Dockerfile-python

```diff
@@ -0,0 +1,20 @@
+FROM python:2.7
+
+WORKDIR /
+
+RUN apt-get update && \
+    apt-get install -y tcpdump libnet1-dev libpcap-dev tar patch wget && \
+    wget "https://github.com/MITRECND/pynids/archive/0.6.2.tar.gz" && \
+    tar -xvzf 0.6.2.tar.gz
+
+WORKDIR /pynids-0.6.2
+
+RUN python setup.py build && python setup.py install
+
+COPY . /app
+
+WORKDIR /app
+
+RUN pip install -r services/requirements.txt
```

Comment:
This is not a big problem, so feel free to ignore this, but would be nice to have a single run also here